{
    "config": {
        "error": {
            "cannot_connect": "Sikertelen csatlakoz\u00e1s",
            "invalid_auth": "\u00c9rv\u00e9nytelen hiteles\u00edt\u00e9s",
            "unknown": "V\u00e1ratlan hiba"
        },
        "step": {
            "user": {
                "data": {
                    "host": "Hoszt",
                    "password": "Jelsz\u00f3",
                    "username": "Felhaszn\u00e1l\u00f3n\u00e9v"
                }
            }
        }
    }
}